src/main/kotlin/racingcar/model/CarRacingGame.kt

```diff
@@ -0,0 +1,39 @@
+package racingcar.model
+
+class CarRacingGame(
+    private val carNames : List<String>?,
+    private val gameRound : String?
+) {
+    private val cars : MutableList<Car> = mutableListOf()
+
+    private val maxGameRound : CarRacingGameMaxRound = CarRacingGameMaxRound(gameRound)
+    private val carRacingGameResults : MutableList<CarRacingGameRoundResult> = mutableListOf()
+
+    init {
+        initSetting()
+    }
+
+    fun calculateRoundResult() : List<CarRacingGameRoundResult>{
+        for (i in 1 .. maxGameRound.carGameMaxRound){
+            val roundResult = CarRacingGameRoundResult(i, cars)
+            carRacingGameResults.add(roundResult)
+        }
+        return carRacingGameResults
+    }
+
+    fun getWinners() : List<String>?{
+        carRacingGameResults.forEach {
+            if (it.round == maxGameRound.carGameMaxRound){
+                return it.getWinners()
+            }
+        }
```

Comment:
it 이 많아지면 이해할 수 없는 경우가 많아요.
```suggestion
        carRacingGameResults.forEach { result ->
            if (result.round == maxGameRound.carGameMaxRound){
                return result.getWinners()
            }
        }
```